sql/group.go

```diff
@@ -178,11 +178,43 @@ func (n *groupNode) wrap(plan planNode) planNode {
 	if n == nil {
 		return plan
 	}
+	// Check to see if the requested ordering is compatible with the existing
+	// ordering.
+	existingOrdering, prefix := plan.Ordering()
+	if log.V(2) {
+		log.Infof("Group: existing=%d (%d) desired=%d", existingOrdering, prefix, n.desiredOrdering)
+	}
+
 	n.plan = plan
 	n.needGroup = true
 	return n
 }
 
+// isNotNullFilter adds as a "col IS NOT NULL" constraint to the expression if
+// the groupNode has a desired ordering on col.
```

Comment:
I wasn't checking for a single min/max expression here, because this is also valid if you have something like `min(k), min(k+1)`. That is, multiple min/max expressions, but all on the same column. I'll expand the comment.